src/Raven.Client/Documents/Commands/Batches/PutAttachmentCommandData.cs

```diff
@@ -9,11 +10,20 @@ namespace Raven.Client.Documents.Commands.Batches
 {
     public sealed class PutAttachmentCommandData : ICommandData
     {
-        public PutAttachmentCommandData(string documentId, string name, Stream stream, string contentType, string changeVector) : this(documentId, name, stream, contentType, changeVector, false)
+        [Obsolete("This constructor is deprecated and will be removed in next RavenDB major version, use the overload with retireAttachmentParameters instead")]
```

Comment:
Why make it Obsolete? Retiring attachment is not mainstream - so we should keep this as a default one.